README.md

```diff
@@ -201,10 +234,40 @@ To include all the files in a directory, use the `*` wild card (eg. `/var/log/*`
 | `hostname`            | no       | `hostname`                     | `myhostname`                   | Hostname |
 | `input_file_path_n`   | no       | `unset`                        | `/var/log/*`                   | Input log file path. **If this variable is not specified, default log agent configuration file is created.** |
 
-Additionally, you can add the `--no_service` to omit the step of automatically creating `monasca-log-agent.service` in `/etc/systemd/system/`
+Additionally, you can add the `--no_service` to omit the step of automatically
+creating `monasca-log-agent.service` in `/etc/systemd/system/`
+
+### Uninstalling Logs agent
+Stop the service:
+```
+systemctl stop monasca-log-agent
+systemctl disable monasca-log-agent
+```
+
+In the following description, `[target_dir]` is the target directory specified
+at the time of running `log-agent-[version].run`.
 
-### Monasca-ui
+Delete obsolete files:
+```
+rm -f [target_dir]/
+rm -f /etc/systemd/system/monasca-log-agent.service
+systemctl daemon-reload
```

Comment:
I checked `systemctl status monasca-log-agent` before and after `systemctl daemon-reload`.
But service isn't removed.
```
# systemctl status monasca-log-agent
Failed to get properties: Access denied
# systemctl daemon-reload
# systemctl status monasca-log-agent
● monasca-log-agent.service
   Loaded: not-found (Reason: No such file or directory)
   Active: failed (Result: signal) since Mon 2018-01-22 15:41:22 JST; 2min 4s ago
 Main PID: 15489 (code=killed, signal=KILL)

Jan 22 15:40:59 <hostname> systemd[1]: Starting monasca-log-agent.service...
Jan 22 15:41:19 <hostname> logstash[15489]: {:timestamp=>"2018-01-22T01:41:19.602000-0500", :mess...rn}
Jan 22 15:41:19 <hostname> logstash[15489]: {:timestamp=>"2018-01-22T01:41:19.624000-0500", :mess...d"}
Jan 22 15:41:22 <hostname> systemd[1]: Stopping monasca-log-agent.service...
Jan 22 15:41:22 <hostname> systemd[1]: monasca-log-agent.service stop-sigterm timed out. Killing.
Jan 22 15:41:22 <hostname> systemd[1]: monasca-log-agent.service still around after SIGKILL. Ignoring.
Jan 22 15:41:22 <hostname> systemd[1]: monasca-log-agent.service: main process exited, code=kill...KILL
Jan 22 15:41:22 <hostname> systemd[1]: Stopped monasca-log-agent.service.
Jan 22 15:41:22 <hostname> systemd[1]: Unit monasca-log-agent.service entered failed state.
Jan 22 15:41:22 <hostname> systemd[1]: monasca-log-agent.service failed.
Hint: Some lines were ellipsized, use -l to show in full.
```

Reply:
I will look into this later.

Reply:
I added `systemctl reset-failed monasca-log-agent` command, should get rid of this problem.